platform-core-module/src/main/java/com/softicar/platform/core/module/localization/AGLocalization.sqml

```diff
@@ -9,6 +9,7 @@ TABLE AGLocalization 'Core.Localization' {
 	AGCoreLanguage language
 	String decimalSeparator [MAXLENGTH=255]
 	String digitGroupSeparator = '' [MAXLENGTH=255]
+	String dateFormat [MAXLENGTH=255]
```

Comment:
I see no adaption of the LocalizationValidator and LocalizationValidatorTest

Reply:
nice catch! forgot about those.

Reply:
...`LocalizationValidator` was wrongfully implemented: It threw exceptions instead of adding errors to the validation results. This led to premature termination of the overall validation when trying to create or edit a record from the UI, where fixing one input error could unlock further input errors.
`LocalizationValidator` was fixed accordingly, which entailed a rewrite of `LocalizationValidatorTest`.

Reply:
![image](https://user-images.githubusercontent.com/15086658/179935348-3e67701f-d052-4048-a329-42f2d636c136.png)

Reply:
nice